src/parsing/include/antioch/parser_base.h

```diff
@@ -343,14 +346,15 @@ namespace Antioch
   }
 
   template <typename NumericType>
-  void ParserBase<NumericType>::not_implemented() const
+  void ParserBase<NumericType>::not_implemented(unsigned int i) const
   {
       std::cerr << "\n*********************************************************\n"
                 << "This method is not available with a " << _type << " parser.\n"
                 << "Parsing file " << _file << ".\n"
                 << "No format has been defined yet.  Maybe contribute?\n"
                 << "https://github.com/libantioch/antioch\n" 
-                << "\n*********************************************************\n"
+                << "\nMethod code: " << i
```

Comment:
Could we switch all these to antioch_not_implemented()?  The error message here is nicer but we could move the generic parts of it to antioch_asserts.h and/or add an antioch_not_implemented_msg() option for non-generic additions.  The "Method code" here is *not* nicer than just getting a filename and line number in the output.